pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-preparing-content-for-publication.adoc

```diff
@@ -0,0 +1,18 @@
+include::../attributes.adoc[]
+
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-preparing-content-for-publication_{context}"]
+= Preparing content for publication
+
+:context: assembly-preparing-content-for-publication
+
+[role="_abstract"]
+Prepare your {ContentTerm} for publication by assigning it metadata. After you assign metadata, you can preview your {ContentTerm} to see how it looks in the {PublishingPortal}. 
```

Comment:
@theashiot 
Be more specific. The way this is phrased, it's not clear that the only things you need to do to prep your content for publication is to assign metadata and preview it in the portal. It kind of begs the question..."Is there anything else I need to do?"

Reply:
How about "After you have imported your content into Pantheon, prepare it for publication by assigning metadata".

Reply:
LGTM!